public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,1087 @@
+  full `a11y` compliance. And this brings us to another extremely important aspect of `a11y` testing.
+  
+:marked
+  ### The screen reader  
+  
+  When creating web pages it would be preposterous to never open it in a browser before release day. 
+  
+  The same is true for screen readers. The tooling already described should be used to speed up our development
+  cycle, but the only way to truly experience our web pages in a screen reader is by using one.
+  
+  However, using a screen reader for the first time can be a daunting task so here we look at three of the most
+  used screen reader / browser combinations and how to get started with them. You do not have to be an expert in
+  screen reader usage to test your websites for `a11y`!
+  
+.l-sub-section
+  :marked
+    You will use the skills you gained in the `keyboard navigation` section when using a screen reader. First get 
+    comfortable with keyboard-only navigation before adding a screen reader to it. This will help reduce the 
+    confusion that first time users of screen readers usually get confronted by.
+
+.l-sub-section
+  :marked
+    You will not get the same experience in every `screen reader / browser combination`, so do not spend too much time
+    trying to fix *"errors"* in combinations not mentioned here. Often you will be trying to fix a technology 
+    integration issue and not a problem on your page. Focus on the three combinations
+    below and you will ensure that the bulk of users out there can use your websites. 
+    
+:marked
+  #### NVDA (Open source screen reader) in Firefox
+  
+  [NonVisual Desktop Access](http://www.nvaccess.org/), better known as `NVDA` is an open source screen reader for
+  `Microsoft Windows`. For best results, use this screen reader with `Mozilla Firefox`.
+  
+  Both `WebAIM` ([Using NVDA to Evaluate Web Accessibility](http://webaim.org/articles/nvda/)) as well as 
+  `The A11y Project` ([Getting Started with NVDA](http://a11yproject.com/posts/getting-started-with-nvda)) have 
+  excellent introductory guides for getting started with `NVDA`. We recommend that you read either one or both of them. 
+  
+:marked
+  #### JAWS (Most used screen reader) in IE 10+
+  
+  [Job Access With Speech](http://www.freedomscientific.com/Products/Blindness/JAWS), better known as `JAWS`, is the 
+  most used screen reader. This is not a free product, but if you are serious about your `a11y` testing, this is not
+  a screen reader you can ignore. It does offer a `40 minute per session` trial which helps the beginner `a11y` tester
+  also check `a11y` in this screen reader.
+  
+  `JAWS` is best used in `Internet Explorer 10+`.
+  
+  For an introduction on using `JAWS`, head over to `WebAIM` and read their article on 
+  [Using JAWS to Evaluate Web Accessibility](http://webaim.org/articles/jaws/).
+  
+:marked
+  #### VoiceOver (Apple OS X integrated screen reader) in Safari
```

Comment:
I was actually thinking about it. But then decided that this was not the place for it. Wanted to home in on a few dev tools that can easily be used during development time, and especially to introduce developers to testing accessibility matters. The mobile screen readers, I feel, is one more step that naturally happens when someone gets into accessibility. Maybe it will show up in a doc in future.